manuscript/chapter3.txt

```diff
@@ -7,29 +7,29 @@ C> By [John Haynes](https://www.youtube.com/channel/UCJVXolxwB4x3EsBAzSACCTg) |
 
 ## Disclaimer & Overview
 
-This chapter is a beginner’s guide on how to crack passwords. While on the surface this may seem to be something reserved for cybercriminals, there are legitimate reasons for a law-abiding individual to understand this process. Firstly, those who work in penetration testing or a red team environment will need to know how to do this task. Secondly, law enforcement may need to access data that is password protected with the legal authority of a search warrant. Third, important data may need to be recovered from a device after the owner is deceased for the estate or heirs. There may also be other ways to legally access password-protected data such as forgotten passwords or security concerns in a corporate environment. Finally, it is important for someone who wishes to keep their data secure to understand this process to know why a strong password is important and how to test the security of their passwords without compromising those passwords.
+What follows is a beginner’s guide to password cracking. While it may seem to be solely the domain of cybercriminals, there are  reasons law-abiding people may wish to understand this process. Red-teamers, law enforcement officers, the next-of-kin of a deceased person, and corporate IT staff, and anyone with legal authorization to access a device or service for which the password is unknown can make use of these techniques. Additionally, an understanding of these principles can help anyone who wishes to keep their own data secure to select strong passwords and test their security safely. None of the information and techniques described here are intended for malicious or illegal uses.
```

Comment:
Minor typo correction, remove the first 'and': 

"...the next-of-kin of a deceased person, corporate IT staff, and anyone with legal authorization to access a device or service for which the password..."